packages/CellOracle/meta.yaml

```
@@ -0,0 +1,21 @@
+name: CellOracle
+description: |
+    A computational tool that integrates single-cell transcriptome and epigenome profiles 
+    to infer gene regulatory networks (GRNs), critical regulators of cell identity.
+project_home: https://github.com/morris-lab/CellOracle
+documentation_home: https://morris-lab.github.io/CellOracle.documentation/
+tutorials_home: https://morris-lab.github.io/CellOracle.documentation/
+publications:
+    - 10.1101/2020.02.17.947416
+install:
+    pypi: celloracle
+tags:
+    - GRN
+    - TF
+license: Apache-2.0
+version: v0.10.12
+authors:
+    - KenjiKamimoto-wustl122
+    - sam-morris
+    - cmhct7
+test_command: null
```

```suggestion
test_command: |
            sudo apt-get update
            sudo apt-get install -y bedtools
            wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
            bash miniconda.sh -b -p $HOME/miniconda
            source "$HOME/miniconda/etc/profile.d/conda.sh"
            hash -r
            conda config --set always_yes yes --set changeps1 no
            #conda update -q conda
            # Useful for debugging any issues with conda
            conda info -a
            # Create env
            conda create -q -n test-environment python=3.8
            # command to install dependencies
            conda activate test-environment
            conda install cython numpy pytest
            
            # Check pip
            echo pip_path
            which pip
            
            pip install .
            # command to run tests
            pytest -v
```

https://github.com/morris-lab/CellOracle/blob/ff5d3e84cff00509e359ca351f4b61c45143a4f0/.github/workflows/build_check.yml